### PR TITLE
added business category multi select

### PR DESCRIPTION
**Pull request by sirLisko**
**What**  
Added dynamic question for category/subcategory.

Screenshot 2020-11-25 at 12 24 13

### CODE REVIEW

File: components/Steps/index.jsx [hunk 1]
```diff
@@ -93,6 +94,19 @@ export const inputLabels = {
...
       adminValidation: true,
     },
+    businessSubCategory: {
+      label: 'Please select the sub-category:',
+      validation: {
+        required: true,
+        validate: (value) => value !== '',
+      },
+    },
+    businessCustomeCategory: {
```
Comment (mattbee) [inline components/Steps/index.jsx:104]: Is this a typo, as per other comment? `businessCustomCategory` perhaps?

File: lib/dbMapping.js [hunk 2]
```diff
@@ -87,3 +87,73 @@ export const APPLICATION_STATE = [
...
+    'Homeware stores',
+    'Tobacco and vape shops',
+    'Electronic goods and mobile phone shops',
+    'Markets (Non-Essential)',
+    'Car and other vehicle showrooms',
+    'Car washes',
+    'Auction houses',
+    'Other Non-Essential Retail',
+  ],
+  Hospitality: ['Restaurants', 'Pubs', 'Bars', 'Cafes', 'Other Hospitality'],
```
Comment (mattbee) [inline lib/dbMapping.js:103]: Really picky but can we make the formatting consistent? i.e. each array item on a new line (took me a little bit to parse this line and realise it was same as the others.)

**Review by mattbee — APPROVED** [review]
Nothing blocking (I feel like I'm being a bit picky, so sorry about that!)

File: components/Steps/EligibilityCriteriaDetails.jsx [hunk 3]
```diff
@@ -55,13 +60,38 @@ const Step1 = (props) => {
...
+                errors
+              )}
+              options={BUSINESS_CATEGORIES[selectedCategory]}
+            />
+          )}
+          {BUSINESS_SUB_WITH_FREETEXT.includes(selectedSubCategory) && (
+            <TextInput
+              {...getInputProps(
+                'eligibilityCriteriaDetails',
+                'businessCustomeCategory',
```
[the remark this answers is not in the document]
Comment (sirLisko) [reply components/Steps/EligibilityCriteriaDetails.jsx]: fixed!

File: lib/dbMapping.js (hunk 2 above)
Comment (sirLisko) [reply lib/dbMapping.js:103]: I know that could be annoying, but we are using https://prettier.io/ for formatting and it's all automatic. They are not in the same format for a matter of print width https://prettier.io/docs/en/rationale.html#print-width :(

**Pull request merged**